stable/yugabyte/templates/_helpers.tpl

```diff
@@ -206,7 +206,7 @@ Get files from fs data directories for readiness / liveness probes.
 Generate server FQDN.
 */}}
 {{- define "yugabyte.server_fqdn" -}}
-  {{- if (and .Values.istioCompatibility.enabled .Values.multicluster.createServicePerPod) -}}
+  {{- if (or .Values.istioCompatibility.enabled .Values.multicluster.createServicePerPod) -}}
```

Comment:
For all these cases you can just drop the `.Values.istioCompatibility.enabled` from this condition, because all are more related to `createServicePerPod` actually.

```yaml
  {{- if .Values.multicluster.createServicePerPod -}}
```